src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -448,7 +450,7 @@ function MoneyRequestView({
                     />
                 )}
                 {!shouldShowReceiptEmptyState && !hasReceipt && <View style={{marginVertical: 6}} />}
-                {shouldShowNotesViolations && <ReceiptAuditMessages notes={receiptViolations} />}
+                {canUseViolations && <ViolationMessages violations={getViolationsForField('receipt')} />}
```

Comment:
That change must have broken something that isn't being tested in the manual tests of this PR (otherwise we would've noticed it broke something). Given that, consider adding a manual test and confirming your changes/code solve for the case in question